Assets/Scripts/Model/RollModifier/InjuryModifier.cs

```diff
@@ -4,16 +4,16 @@ public class InjuryModifier : AbstractModifier
     {
         public InjuryModifier(string name, int modifier) : base(name, modifier) { }
 
-        public bool NigglingInjuryModifier { get; set; }
+        public override ShowModifierSignMode ShowModifierSign { get; set; } = ShowModifierSignMode.ShowReversed;
```

Comment:
Seems excessive to override this property to change the default value. Set the default in the constructor instead.